plugins/docsettingtweak.koplugin/directory_defaults.lua

```diff
@@ -1,9 +1,11 @@
 return {--do NOT change this line
 
  --HELP:
- -- you can override default settings for documents per directory in this file
- -- the directories must be under the home folder.
- -- you can find settings to change in the metadata.lua in the .sdr folder.
+ -- You can override default settings for documents per directory in this file.
+ -- The directories must be under the home folder.
+ -- You can find settings to change in the metadata.lua in the .sdr folder.
+ -- The path must start with "/" (absolute path).
+ -- The path may not end with a "/" it must end with the folder name.
```

Comment:
"_must_ not end with a /" then? ("_may not_" sounds to me like it would be allowed).